### PR TITLE
Scroll into view

### PR DESCRIPTION
**Pull request by fstoerkle**

### CODE REVIEW

File: src/ui/box/global_message.jsx [hunk 1]
```diff
@@ -15,5 +30,10 @@ export default class GlobalMessage extends React.Component {
 
 GlobalMessage.propTypes = {
   message: PropTypes.node.isRequired,
-  type: PropTypes.oneOf(['error', 'success']).isRequired
+  type: PropTypes.oneOf(['error', 'success']).isRequired,
+  scrollIntoView: PropTypes.bool
```
Comment (fstoerkle) [inline src/ui/box/global_message.jsx:34]: shouldn't this be `isRequired`?

File: src/ui/box/container.jsx [hunk 2]
```diff
@@ -258,7 +260,10 @@ export const defaultProps = (Container.defaultProps = {
   disableSubmitButton: false,
   isMobile: false,
   isSubmitting: false,
-  logo: `${isFileProtocol ? 'https:' : ''}//cdn.auth0.com/styleguide/components/1.0.8/media/logos/img/badge.png`,
+  logo: `${isFileProtocol
+    ? 'https:'
+    : ''}//cdn.auth0.com/styleguide/components/1.0.8/media/logos/img/badge.png`,
```
Comment (fstoerkle) [inline src/ui/box/container.jsx:265]: Is this unrelated change necessary?

File: src/ui/box/container.jsx [hunk 3]
```diff
@@ -26,13 +26,12 @@ const badgeSvg = (
...
 
-const BottomBadge = ({ link }) => (
+const BottomBadge = ({ link }) =>
   <span className="auth0-lock-badge-bottom">
     <a href={link} target="_blank" className="auth0-lock-badge">
       Protected with {badgeSvg}
     </a>
-  </span>
-);
+  </span>;
```
Comment (fstoerkle) [inline src/ui/box/container.jsx:34]: Is this unrelated change necessary, or this is just `prettier`?

File: src/__tests__/ui/box/global_message.test.jsx [hunk 4]
```diff
@@ -12,6 +12,52 @@ describe('GlobalMessage', () => {
...
+    const scrollIntoViewSpy = jest.fn();
+    wrapper.getDOMNode().getBoundingClientRect = getBoundingClientRectSpy;
+    wrapper.getDOMNode().scrollIntoView = scrollIntoViewSpy;
+
+    wrapper.getNode().componentDidMount();
+
+    expect(scrollIntoViewSpy).not.toHaveBeenCalled();
+  });
+  it('should not call scrollIntoView if parameter is set to false', () => {
+    const wrapper = mount(<GlobalMessage type="success" message="foo" />);
```
Comment (fstoerkle) [inline src/__tests__/ui/box/global_message.test.jsx]: actually, the parameter is not set to `false` here

**Comment by fstoerkle** [comment]
I would suggest to add the new config option to the documentation as well

File: src/ui/box/global_message.jsx (hunk 1 above)
Comment (m-idler) [reply src/ui/box/global_message.jsx:34]: We don't wanted this to be a breaking change, so we made it optional

File: src/ui/box/container.jsx (hunk 2 above)
Comment (m-idler) [reply src/ui/box/container.jsx:265]: is formatted automatically by prettier

File: src/ui/box/container.jsx (hunk 3 above)
Comment (m-idler) [reply src/ui/box/container.jsx:34]: removed the unnecessary brackets

**Comment by fstoerkle** [comment]
LGTM
